src/file_validator.py

```diff
@@ -234,8 +234,14 @@ def validate_file(self, fileRecord):
             }
             self.mongo_dao.save_file_md5(md5_info)
 
-        if int(org_size) != int(size) or org_md5 != md5:
-            msg = f'File “{file_name}”: file integrity check failed.'
+        if int(org_size) != int(size):
+            msg = f'File “{file_name}”: expected size: {org_size}, actual size: {size}.'
+            self.log.error(msg)
+            error = create_error("Data file Integrity check failed", msg)
```

Comment:
Please also change the title of the error, it's in the comment in the ticket

Reply:
updated